<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialSetupProjet</class>
 <widget class="QTabWidget" name="DialSetupProjet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="tabPosition">
   <enum>QTabWidget::West</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="currentIndex">
   <number>1</number>
  </property>
  <widget class="QWidget" name="comptesTab">
   <attribute name="title">
    <string>Comptes</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Comptes</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="compteView">
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Nom</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Somme</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Monnaie</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addCompte">
           <property name="text">
            <string>Nouveau</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteCompte">
           <property name="text">
            <string>Supprimer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="budgetTab">
   <attribute name="title">
    <string>Budget</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_Apercu">
      <property name="title">
       <string>Apercu</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGraphicsView" name="apercu">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="backgroundBrush">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>119</red>
            <green>128</green>
            <blue>245</blue>
           </color>
          </brush>
         </property>
         <property name="interactive">
          <bool>true</bool>
         </property>
         <property name="sceneRect">
          <rectf>
           <x>0.000000000000000</x>
           <y>0.000000000000000</y>
           <width>100.000000000000000</width>
           <height>100.000000000000000</height>
          </rectf>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="rubriqueProperty">
         <property name="title">
          <string>Propriétés</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="formAlignment">
             <set>Qt::AlignCenter</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="nomLabel">
              <property name="text">
               <string>Nom:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="rubriqueNom"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="couleurLabel">
              <property name="text">
               <string>Couleur:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QToolButton" name="rubriqueColor">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 223, 93);</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="iconSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="soldeMensuelLabel">
              <property name="text">
               <string>Solde mensuel:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="GSpinBox" name="rubrqueSolde">
              <property name="accelerated">
               <bool>false</bool>
              </property>
              <property name="prefix">
               <string/>
              </property>
              <property name="suffix">
               <string> €</string>
              </property>
              <property name="maximum">
               <double>10000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Rubriques</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTreeWidget" name="rubriquesView">
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::MoveAction</enum>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Nom</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Solde</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>test</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addRubrique">
           <property name="text">
            <string>New</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addSubRubrique">
           <property name="text">
            <string>New Sub</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteRubrique">
           <property name="text">
            <string>Supr</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="advancedTab">
   <attribute name="title">
    <string>Avancé</string>
   </attribute>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QListWidget" name="advanceSelector">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>175</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <item>
       <property name="text">
        <string>Types Compte</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Monnaies</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="advancedStackedView">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pageTypeCompte">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="typeCompteWidget">
          <property name="title">
           <string>Types Compte</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageMonnaie">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGroupBox" name="monnaieWidget">
          <property name="title">
           <string>Monnaies</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>GUI/gspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>advanceSelector</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>advancedStackedView</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
